### DISCUSSION TITLE
github copilot in renewed membership

### DISCUSSION THREAD

**Discussion by kaavyabaranwal** [community/community#144637] [GitHub Education]
### Hi everyone,

Just renewed my github education membership. Does my github copilot also gt renewed?

**Comment by davevad93** [comment]
Hi @kaavyabaranwal ,

Yes, when you renew the `Student Developer Pack` you have access to all the offers included in the [pack](https://education.github.com/pack) e.g. **free access to Copilot Individual while you're a student**.
